test/test-style.js

```diff
@@ -136,23 +136,23 @@ function processData(filename, logger) {
   /** @type {import('../types').CompatData} */
   let dataObject = JSON.parse(actual);
   let expected = JSON.stringify(dataObject, null, 2);
-  let expectedSorting = JSON.stringify(dataObject, orderSupportBlock, 2);
+  let expectedBrowserSorting = JSON.stringify(dataObject, orderSupportBlock, 2);
 
   // prevent false positives from git.core.autocrlf on Windows
   if (IS_WINDOWS) {
     actual = actual.replace(/\r/g, '');
     expected = expected.replace(/\r/g, '');
-    expectedSorting = expectedSorting.replace(/\r/g, '');
+    expectedBrowserSorting = expectedBrowserSorting.replace(/\r/g, '');
   }
 
   if (actual !== expected) {
     hasErrors = true;
     logger.error(chalk`{red Error on {bold line ${jsonDiff(actual, expected)}}}`);
   }
 
-  if (expected !== expectedSorting) {
+  if (expected !== expectedBrowserSorting) {
     hasErrors = true;
-    logger.error(chalk`{red Browser sorting error on {bold line ${jsonDiff(expected, expectedSorting)}}}`);
+    logger.error(chalk`{red Browser sorting error on {bold line ${jsonDiff(expected, expectedBrowserSorting)}}});
```

Comment:
Sorry, I introduced a new problem

```suggestion
    logger.error(chalk`{red Browser sorting error on {bold line ${jsonDiff(expected, expectedBrowserSorting)}}}`);
```